src/libs/actions/Plaid.js

```diff
@@ -27,15 +27,22 @@ function openPlaidBankAccountSelector(publicToken, bankName, allowDebit) {
         allowDebit,
         bank: bankName,
     }, {
-        optimisticData: [{
-            onyxMethod: CONST.ONYX.METHOD.MERGE,
-            key: ONYXKEYS.PLAID_DATA,
-            value: {
-                isLoading: true,
-                error: '',
-                bankName,
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.PLAID_DATA,
+                value: {
+                    isLoading: true,
+                    error: '',
+                    bankName,
+                },
             },
-        }],
+            {
+                onyxMethod: CONST.ONYX.METHOD.SET,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT,
+                value: null,
+            },
```

Comment:
Maybe it would make more sense to clear the REIMBURSEMENT_ACCOUNT_DRAFT key when calling [openPlaidBankLogin](https://github.com/Expensify/App/blob/2483ad0a692b1392d7dc79f917edd44c5be46080/src/libs/actions/Plaid.js#L12).

```js
API.read('OpenPlaidBankLogin', params, {
    optimisticData: [
        {
            onyxMethod: CONST.ONYX.METHOD.SET,
            key: ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT,
            value: null,
        },
    ],
});
```

cc: @nkuoch Since you're touching this code in https://github.com/Expensify/App/pull/13236/files#diff-bf2d7acc015c7abd6aa970a11d183001b238e66806abbcaaa6d010ce4439cfd6R16-R26

Reply:
Indeed, it will bed fixed by https://github.com/Expensify/App/pull/13668